pages/[id].tsx

```diff
@@ -76,19 +74,23 @@ const CompanyResult = ({ company }: { company: Company }) => {
 };
 
 export const getStaticProps: GetStaticProps = async ({ params }) => {
-  const vatNumber = params?.id as string;
+  try {
+    const vatNumber = params?.id as string;
 
-  const hoc = databaseHoc()(async (req) => {
-    const { companies } = await getCompanies(req.db, 1, 0, vatNumber);
-    return companies;
-  });
-  const companies = await hoc({} as NextRequestWithDb, {} as NextApiResponse);
-
-  return {
-    props: {
-      company: companies[0],
-    },
-  };
+    const res = await fetch(`https://sertifikaattilukija.herokuapp.com/data?name=${vatNumber}`);
+    const { data: companies } = await res.json();
+    return {
+      props: {
+        company: companies[0],
+      },
+    };
+  } catch (err) {
+    return {
+      props: {
+        error: err,
+      },
+    };
```

Comment:
Most likely error is a 404 somewhere in the process so I'd return
`    { notFound: true };
`
I'd also check that companies variable exists and is non empty